### PR TITLE
Create Contribution.md

### PR DESCRIPTION
**Pull request by Parth4git**
Solve the issue #344
Kindly review my PR. @Licenser

### CODE REVIEW

File: Contribution.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+- Aim for code that is efficient and idiomatic in Rust.
+- Ensure your code is compatible with serde for JSON serialization/deserialization.
+
+## Testing
+
+We require comprehensive test coverage to maintain the quality of the codebase. Write unit tests for your code and ensure that existing tests pass. Use continuous integration tools to automatically run tests for various Rust versions.
+
+## Documentation
+
+Documentation is crucial for helping others understand the project.There are bindings for upstream simdjson available [Here](https://crates.io/crates/simdjson-rust) this includes the README, Dependencies, and any other relevant documentation files.
```
Comment (Licenser) [inline Contribution.md:72]: The documentation of the wrapper crate doesn't have much value for this section.

I am for linking to alternatives, but that feels more at home in the `README.md` than in the `CONTRIBUTING.md`. And we already [have a link to simdjson-rust](https://github.com/simd-lite/simd-json#other-interesting-things) there.

File: Contribution.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+### Code Contributions
+
+For code contributions:
+
+1. Fork the project repository on GitHub.
+2. Clone your fork locally: `git clone https://github.com/your-username/your-repo.git`
+3. Create a new branch for your changes: `git checkout -b feature/your-feature`
+4. Write your code, following the Code of Conduct (see [Code of Conduct](#CODE_OF_CONDUCT.md)).
+5. Write unit tests for your code (see [Tests](#tests)).
+6. you can refer to `data`folder for further information (see [data](#data))
```
Comment (Licenser) [inline Contribution.md:53]:
```suggestion
6. you can refer to `data` folder for test data already in use.
```

**Review by Licenser — CHANGES_REQUESTED** [review]
I quite like it 👍 I've two minor suggestions in the file.

File: Contribution.md (hunk 2 above)
Comment (Parth4git) [reply Contribution.md]: I have implemented your suggestions in the file. Now, you can review it. @Licenser

**Review by Licenser — APPROVED** [review]
👍 awesome :) It looks great I really appreciate the effort to improve the projects community!

**Pull request merged**